modules/table/src/integrationTest/java/org/apache/ignite/distributed/ItTxDistributedTestSingleNode.java

```diff
@@ -398,14 +399,13 @@ protected Int2ObjectOpenHashMap<RaftGroupService> startTable(String name, UUID t
                         raftSvc -> {
                             try {
                                 replicaManagers.get(node).startReplica(
-                                        grpId,
+                                        new TablePartitionId(tblId, partId),
```

Comment:
I don't like an idea that in some places we use ReplicationGroupId and in some old fashioned String grpId, e.g. grpId on line 387
```
                CompletableFuture<Void> partitionReadyFuture = raftServers.get(node).prepareRaftGroup(
                        grpId,
```

Reply:
It rather connected with the replication layer is dedicated from RAFT protocol implementation. They both just cannot have the same class for identification replica.

Reply:
Why? What if you'll move move Replication to ignite-core? Semantically we use exactly the same entities to identify groups both in replicator and raft. In case of tables it's tableId + partitionIndex. However, because of String nature of in-raft ids it's required to have conversion rules, potentially both sides and explicit grpId params here and there which looks not that good.